### ISSUE TITLE
Create Migrations from Drizzle to Postgres/SqlLite

### ISSUE THREAD

**Issue by arghasen** [CricketManager/CricketManager#34]
Create migrations for database sql schemas in either based on Postgres/sqlite based on what we choose.
Blocked by #6 
Blocked by #33

**Comment by arghasen** [comment]
Drizzle migrations are ready. Closing for now. will open a new issue when database schema is stable
